elements/pfe-navigation/demo/single-nav.html

```diff
@@ -43,7 +43,7 @@
     ])</script>
   </head>
   <body unresolved style="background-color: #ddd;">
-     <pfe-navigation id="pfe-navigation-2">
```

Comment:
Suggestion: You can add `role="navigation"` to pfe-navigation, still get the semantic value of the navigation tag but with less mark-up for the author to have to remember to put on their page.

Reply:
That's a good thought, I'm pretty sure I need the wrapper for non-JS version layout, so might as well make it a `<nav>` tag. I'll double check though.